lib/handlers/injector-sor.ts

```diff
@@ -161,18 +162,23 @@ export abstract class InjectorSOR<Router, QueryParams> extends Injector<
               break
           }
 
-          const provider = new DefaultEVMClient({
-            allProviders: [
-              new InstrumentedEVMProvider({
-                url: {
-                  url: url,
-                  timeout,
-                },
-                network: chainId,
-                name: deriveProviderName(url),
-              }),
-            ],
-          }).getProvider()
+          let provider: StaticJsonRpcProvider
+          if (GlobalRpcProviders.getGlobalUniRpcProviders(log).has(chainId)) {
```

Comment:
we probably want to add a feature flag here instead of rolling it out immediately. WDYT?

Reply:
Now the flag feature has been added. Please see `app.ts` as well as `GlobalRpcProviders.ts`.